core/src/test/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreReaderThreadTest.java

```diff
@@ -67,4 +68,14 @@ public void testWaitUntilOffset() throws Exception {
       fail("5 seconds should be more than enough time to reach offset 0 in the log.");
     }
   }
+
+
+  @Test
+  public void testGroupIdDefault() throws Exception {
```

Comment:
Why did you decide to add default test in one file and configuration test in another?

I almost commented that one test is missing :)